crates/stackable-operator/src/lib.rs

```diff
@@ -22,6 +22,7 @@ pub mod kvp;
 pub mod logging;
 pub mod memory;
 pub mod namespace;
+pub mod patchinator;
```

Comment:
question: Maybe `object_overrides` would be a better name?

Reply:
I really like the "cleverness" of the code to apply arbitrary patches to any k8s resource.
To me personally these are Kubernetes resources (but it's also fine to call them objects) and a list of patches.
I see that we need to call it objectOverrides in the context of SDP because of consistency, but would prefer to keep the code generic, so that we (or technically someone else) might be able to use it for other stuff in the future.

And yeah, I also like the name ^^